src/magento2/application/skeleton/behat.yml.twig

```diff
@@ -1,40 +1,10 @@
 default:
   suites:
-    sample:
-      contexts_services:
-        - web.context
+    application:
+      autowire: true
+      contexts:
+        - Inviqa\Acceptance\Context\BehatConfigurationVerificationContext
+      services: '@bex.magento2_extension.service_container'
+  
   extensions:
-    Behat\MinkExtension:
-      base_url: 'https://{{ @('hostname') }}'
-      default_session: goutte
-      javascript_session: chrome
-      sessions:
-        goutte:
-          goutte:
-            guzzle_parameters:
-              verify: false
-        chrome:
-          chrome:
-            api_url: "http://chrome:9222"
-            validate_certificate: false
-        local_chrome:
-          chrome:
-            api_url: "http://host.docker.internal:9222"
-    Bex\Behat\Magento2InitExtension:
-      magento_configs:
-        - path: admin/security/use_form_key
-          value: 0
-    SensioLabs\Behat\PageObjectExtension:
-      namespaces:
-        page: [Acceptance\Page]
-    FriendsOfBehat\ContextServiceExtension:
-      imports:
-        - 'features/bootstrap/_config/contexts.yml'
-    FriendsOfBehat\CrossContainerExtension: ~
-    Cjm\Behat\StepThroughExtension: ~
-    DMore\ChromeExtension\Behat\ServiceContainer\ChromeExtension: ~
-local: # (google-chrome-stable --remote-debugging-port=9221 &) && socat tcp-l:9222,fork TCP4:127.0.0.1:9221
```

Comment:
We should keep this config

Reply:
okay, I had no idea what are those :D (removed it together with all the browser based stuff)

Reply:
done, Mink is available again, tested and works with goutte and chrome as well